includes/import-functions.php

```diff
@@ -129,10 +129,22 @@ function give_import_get_form_data_from_csv( $data, $import_setting = array() )
 
 				if ( ! empty( $prices ) && is_array( $prices ) && ! empty( $prices[0] ) ) {
 					$prices = wp_parse_args( $multi_level_donations, $prices );
+
```

Comment:
@raftaar1191 I ask you to always create any branch for the issue from latest release branch. Why already merge code is in this PR.
https://github.com/WordImpress/Give/pull/2150/files#diff-70cbfac22d1acc82c0f80a655ac6d0c9